ksqldb-engine/src/main/java/io/confluent/ksql/engine/QueryCleanupService.java

```diff
@@ -106,12 +106,13 @@ public QueryCleanupTask(
         final Optional<String> topologyName,
```

Comment:
super nit: rename to `queryId`? Ksql devs may not necessarily know what it means for a topology to be named or understand  how it relates/why we would even care about this name in the first place